src/index.js

```diff
@@ -128,7 +128,7 @@ class Game extends React.Component {
                 <div className="game-info">
                     <div>{status}</div>
                     <button
-                        onClick={() => this.setState({ orderMovesListAscending: (order ? false : true) })}
+                        onClick={() => this.setState({ orderMovesListAscending: !this.state.orderMovesListAscending })}
```

Comment:
Ou bien : 
onClick={() => this.setState({ orderMovesListAscending: !order })}

Reply:
oui ça serait beaucoup mieux